### PR TITLE
Add private registries test

### PR DESCRIPTION
**Pull request by vladkravets**

### CODE REVIEW

File: tests/src/privateregistries/scripts/privateregistries_tests.py [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+        self.access_token = None
+        self.project_id = None
+
+    def setUp(self):
+        print("Waiting for Artifactory ...")
+        time.sleep(self.await_time_in_secs)
+        for i in range(1, 20):
+            try:
+                response = requests.get(f"{self.artifactory_base_url}/api/system/ping", verify=False)
```
Comment (dbolharovcmx) [inline tests/src/privateregistries/scripts/privateregistries_tests.py:33]: In Jenkins Logs I could see `ArtifactoryClient::http [get] request to /api/system/ping failed with 404 code` is that ok?

Comment (dbolharovcmx) [reply tests/src/privateregistries/scripts/privateregistries_tests.py:33]: Also found `ArtifactoryClient::http [get] request to /api/system/ping failed with 503 code`

Comment (vladkravets) [reply tests/src/privateregistries/scripts/privateregistries_tests.py:33]: yes, it should be fine; it checks itself while starting

File: tests/src/privateregistries/scripts/privateregistries_tests.py [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+            response = requests.put(f"{self.artifactory_base_url}/example-repo-local/com/checkmarx/parent-project/1.0.0/parent-project-1.0.0.pom", data=file_to_send, auth=('admin', 'password'), verify=False)
+            print(f"Artifactory push pom result: `{response.status_code}`")
+            self.assertEqual(response.status_code, 201)
+
+        # Sign in
+        request_body = {
+            "grant_type": "password",
+            "client_id": "sca_resource_owner",
+            "scope": "sca_api",
+            "acr_values": f"Tenant:{os.environ.get('TEST_SCA_TENANT') or 'lumo'}",
```
Comment (dbolharovcmx) [inline tests/src/privateregistries/scripts/privateregistries_tests.py:76]: oops > credentials

File: tests/src/privateregistries/scripts/privateregistries_tests.py (hunk 1 above)
Comment (vladkravets) [reply tests/src/privateregistries/scripts/privateregistries_tests.py:33]: these are logs of artifactory

File: tests/src/privateregistries/scripts/privateregistries_tests.py [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+            "username": f"{os.environ.get('TEST_SCA_USERNAME')}",
+            "password": f"{os.environ.get('TEST_SCA_PASSWORD')}"
+        }
+        response = requests.post(f"{self.localhost}/identity/connect/token", headers={"Content-Type": "application/x-www-form-urlencoded"}, data=request_body, verify=False)
+        self.assertEqual(response.status_code, 200)
+        self.access_token = response.json()["access_token"]
+
+        # Create a project
+        request_body = {
+            "name": f"ScaAgentTest-{uuid.uuid4().hex}"
```
Comment (dbolharovcmx) [inline tests/src/privateregistries/scripts/privateregistries_tests.py:86]: I think the name of Project can be static, so we don't really add lots of data to our database

Comment (vladkravets) [reply tests/src/privateregistries/scripts/privateregistries_tests.py:86]: project is being removed during the teardown stage

**Review by dbolharovcmx — APPROVED** [review]
Good Job

**Comment by vladkravets** [comment]
[Add private registries tests to the agent](https://trello.com/c/JyN27mUJ/1098-add-private-registries-tests-to-the-agent)

**Pull request merged**